Source/KSCrash/Source/KSCrash/Recording/Sentry/BSG_KSCrashSentry_Signal.c

```diff
@@ -107,7 +107,16 @@ void bsg_kssighndl_i_handleSignal(int sigNum, siginfo_t *signalInfo,
         bsg_g_context->signal.signalInfo = signalInfo;
 
         BSG_KSLOG_DEBUG("Calling main crash handler.");
-        bsg_g_context->onCrash();
+        char errorClass[21];
+        const char *sigName = bsg_kssignal_signalName(sigNum);
```

Comment:
the return value of `sigName` can be NULL, could that cause problems here?

Reply:
yes